### PR TITLE
Editor can sign in

### PR DESCRIPTION
**Pull request by aleciabenjamin**
#### Description
As an Editor
In order to edit and review articles
I would like to log-in as an Editor

Changes proposed in this pull request:
* Assign editor role to allow access to the editor's employee page

#### What I have learned working on this feature:
I have a greater understanding of namespace and have more experience in assigning roles.
#### Screenshots:
Screenshot 2019-03-21 at 15 29 59

### CODE REVIEW

File: features/editor_can_sign_in.feature [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+	Scenario: Logging in as Journalist (access to staff page)
+		Given I am logged in as "amanda@mail.com"
+		And I visit the "landing" page
+		When I click on "Employee"
+		Then I should see "Welcome to the staff page"
+
+	Scenario: Logging in as User (not access to staff page)
+		Given I am logged in as "alecia@mail.com"
+		And I visit the "landing" page
```
Comment (nevroje) [inline features/editor_can_sign_in.feature]: Maybe we can use these 3 steps "And I visit the "landing" page as a background

Comment (aleciabenjamin) [reply features/editor_can_sign_in.feature]: As in "And they visit the landing page"?  Could you be more specific, please:)

File: app/views/layouts/_navbar.html.haml [hunk 2]
```diff
@@ -6,8 +6,9 @@
 			= link_to_unless_current "English", locale: "en", class: "nav_links"
 			= link_to_unless_current "Svenska", locale: "sv", class: "nav_links"
 		.subscription
-			-if(user_signed_in? && current_user.role == "journalist")
+			-if(user_signed_in? && (current_user.role == "journalist" || current_user.role == "editor"))
 				= link_to 'Employee', staff_content_articles_path, class: "subscribe_button"
+			-elsif(user_signed_in?)
```
Comment (oliverochman) [inline app/views/layouts/_navbar.html.haml]: refactor this to
```
-elseif(!user_signed_in?)
  = link_to t('login'), new_user_session_path, class: "subscribe_button"
  = link_to t('signup'), new_user_registration_path, class: "subscribe_button"
```

Comment (aleciabenjamin) [reply app/views/layouts/_navbar.html.haml]: okie doke.

Comment (LauraRe) [reply app/views/layouts/_navbar.html.haml]: done

Comment (aleciabenjamin) [reply app/views/layouts/_navbar.html.haml]: done

**Pull request merged**